HHS/Head-Start-TTADP
# [TTAHUB-1687] Remove goal type field from display on website

## Description of change
We will no longer collect "Goal Type" or specify whether a goal is a RTTAPA goal or not throughout the interface. It has been removed from the frontend entirely. Historical data will be retained; goals simply won't be updated any longer and the field won't retain any meaning.

## How to test
Scour the frontend and ensure that there is no evidence of the RTTAPA badges/Goal type form field, and that you have no issue creating, editing, and viewing goals.

## Issue(s)
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1687

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
